### PR TITLE
[DO-2104] fixed conversion to float

### PR DESCRIPTION
**Pull request by EmmaSamoilova**

### CODE REVIEW

File: business_register/converter/declaration.py [hunk 1]
```diff
@@ -87,7 +87,11 @@ def log_error(self, message):
 
     def to_float(self, value):
         if value not in self.NO_DATA:
-            return float(value.replace(',', '.').strip('.,'))
+            value = value.replace(',', '.').strip('.,')
+            if len(value.split('.')[0]) <= 10:
+                return float(value)
+            else:
+                self.log_error(f'Too many value for valuation = {value}')
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py:94]: We cannot understand the problem from only the value).

**Review by IegorSoboliev — APPROVED** [review]

File: business_register/converter/declaration.py (hunk 1 above)
Comment (EmmaSamoilova) [reply business_register/converter/declaration.py]: done

**Review by criticus — APPROVED** [review]

**Review by OlexandrTopuzov — APPROVED** [review]

**Review by Andrii17s — APPROVED** [review]

File: business_register/converter/declaration.py [hunk 2]
```diff
@@ -1428,58 +1336,7 @@ def save_securities(self, securities_data, declaration):
...
-                acquisition_date=acquisition_date,
-                share=share,
-                pep=pep,
-                company=company,
-                # TODO: decide should we use lower() for storing names
-                full_name=full_name,
-                company_name=company_name,
-                owner_type=owner_type,
-            )
+                self.save_right(securities, data)
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py]: we should call this method in all cases (for 'person' field also)

File: business_register/converter/declaration.py [hunk 3]
```diff
@@ -1528,52 +1385,7 @@ def save_vehicle(self, vehicles_data, declaration):
...
-                type=type,
-                acquisition_date=acquisition_date,
-                share=share,
-                pep=pep,
-                full_name=full_name,
-                company=company,
-                owner_type=owner_type,
-                company_name=company_name,
-            )
+                self.save_right(vehicle, data)
```
Comment (IegorSoboliev) [inline business_register/converter/declaration.py]: same here and everywhere with a separate right model

**Pull request merged**